buildSrc/src/main/groovy/Classpaths.groovy

```diff
@@ -289,7 +289,7 @@ class Classpaths {
 
     static void inheritParquetHadoop(Project p, String configName = JavaPlugin.IMPLEMENTATION_CONFIGURATION_NAME) {
         Configuration config = p.configurations.getByName(configName)
-        addDependency(config, 'org.apache.parquet', 'parquet-hadoop', '1.13.1')
+        addDependency(config, 'org.apache.parquet', 'parquet-hadoop', '1.14.0')
```

Comment:
We should investigate what the release notes means for:

[PARQUET-1822](https://issues.apache.org/jira/browse/PARQUET-1822) - Parquet without Hadoop dependencies

that would be quite nice if we don't need some of the hadoop deps.

Reply:
I was looking at the linked changes and the main commit seems like this: https://github.com/apache/parquet-mr/pull/1141/files#diff-b044ae9879a94e2b8a49d6e6911ea5498ef162df1373cc049ded6256980a7248

One interesting class they have now added are `org.apache.parquet.conf.PlainParquetConfiguration` to replace  `org.apache.hadoop.conf.Configuration`. 

Some other interesting classes that I noticed are
- `org.apache.parquet.hadoop.CodecFactory` which can potentially replace the usage of `org.apache.hadoop.io.compress.CompressionCodecFactory`.
- `org.apache.parquet.hadoop.CodecFactory.HeapBytesCompressor` which can replace `org.apache.hadoop.io.compress.Compressor`
- `org.apache.parquet.hadoop.CodecFactory.HeapBytesDecompressor` which can replace `org.apache.hadoop.io.compress.Decompressor`

I think there is enough here so we can get rid of the `hadoop-common` dependencies. We would still need `parquet-hadoop` though which internally would be using `hadoop-common` only.
Do you think that would be beneficial?

Reply:
I think so, yes.

Reply:
Sounds good. Is it okay if I make a separate issue for this? 
This would require some dedicated effort for the change as well as for the benchmarking.

Reply:
Created this issue: https://github.com/deephaven/deephaven-core/issues/5517